src/lib/flows/rpgf-upload-results-flow/upload-results.svelte

```diff
@@ -0,0 +1,246 @@
+<script lang="ts">
+  import { invalidateAll } from '$app/navigation';
+  import Button from '$lib/components/button/button.svelte';
+  import Divider from '$lib/components/divider/divider.svelte';
+  import DropZone from '$lib/components/drop-zone/drop-zone.svelte';
+  import CrossCircle from '$lib/components/icons/CrossCircle.svelte';
+  import FileCSV from '$lib/components/icons/FileCSV.svelte';
+  import FileXLSX from '$lib/components/icons/FileXLSX.svelte';
+  import Sharrow from '$lib/components/icons/Sharrow.svelte';
+  import StepHeader from '$lib/components/step-header/step-header.svelte';
+  import StepLayout from '$lib/components/step-layout/step-layout.svelte';
+  import type { StepComponentEvents } from '$lib/components/stepper/types';
+  import downloadUrl from '$lib/utils/download-url';
+  import {
+    getApplicationsCsv,
+    getApplicationsXlsx,
+    importResultsAsCsv,
+    importResultsAsXlsx,
+  } from '$lib/utils/rpgf/rpgf';
+  import type { Round } from '$lib/utils/rpgf/types/round';
+  import { createEventDispatcher } from 'svelte';
+
+  const dispatch = createEventDispatcher<StepComponentEvents>();
+
+  export let round: Round;
+
+  let downloading = false;
+
+  async function handleDownload(format: 'csv' | 'xlsx') {
+    downloading = true;
+
+    const content: Blob | string =
+      format === 'csv'
```

Comment:
The magic strings 'csv' and 'xlsx' are making me a little nervous. Might be nice to use an Enum or constant or something for type safety, etc.?